book/integration-service.md

```diff
@@ -0,0 +1,271 @@
+- [GitOps Service](./gitops-service.md)
+  - Provides the facility to create
+    - Snapshots used to test Builds
+    - Environment to test the Application on
+    - SnapshotEnvironmentBindings to have the Snapshot deployed to a specific Environment
+- [Hybrid Application Service](./hybrid-application-service.md)
+  - Provides the Application and Component model. Integration Service updates the pullspec reference on the Component CR when a component passes its required testing.
+- [Release Service](./release-service.md)
+  - Provides the ReleasePlan that will be used to determine if integration-service should create a Release
+- [Enterprise Contract Service](./enterprise-contract.md)
+  - Provides facilities to validate whether content has passed the Enterprise Contract.
+
+
+## Terminology
+
+**Prelim Pipeline** - This is the test pipeline run for the Component of an Application that gets triggered by a completed Build pipeline. Deprecated name, refer to the Component Pipeline
+
+**Component Pipeline** - This is the same as the Prelim pipeline. Going forward the Prelim Pipeline will be referred to as Component Pipeline
+
+**Final Pipeline** - This is the pipeline run that runs if two or more Components have completed and passed their Component pipeline concurrently so that they should be tested together. Deprecated name, refer to Composite Pipeline.
+
+**Composite Pipeline** - This is the same as the Final Pipeline. Going forward the Final Pipeline will be referred to as Composite Pipeline
+
+**Global Candidate List** - The list of all Component digest that have passed individual component testing. This can be retrieved from the Application to see what Components it is made of and then querying each of the Components `spec.containerImage`. 
+
+**Snapshot** - The list that maps all Components of an Application to their Component Image digests. Once created, the snapshot is immutable. 
+
+**Snasphot Digest** - The snapshot itself should have an identifier constructed from its contents - using a digest of the contents likely makes sense.
+
+**Integration Service** - The service that uses the pipeline, snapshot and environment controllers to watch for Component builds, trigger and manage the testing pipelines and create releases based on the testing outcome(s).
+
+**IntegrationTestScenario** - The custom resource that describes separate PipelineRuns that are supposed to be run by the snapshot controller. Any number of them can be set by the user and each of them includes information about the Tekton pipeline that will be executed - Tekton bundle, pipeline name and parameters, and the Environment details. The Integration Test Scenarios can also specify the contexts where they will be applied - this can be used to define when the scenario will run (Component or Composite stage - or both). If more than one Integration Test Scenario has the same context, their respective PipelineRuns will be executed in parallel.
```

Comment:
Is this the main way that users will interface with/configure the integration service? You say that any number of "them" (I assume you mean the defined PipelineRuns, but it would be good to clarify this) can be set by the user -- is this the only customization that a user will have when interacting with the service?

Reply:
Yes, configuring Integration test scenarios is the only direct way to interact with the service from the user's side.
We can change the `Any number of them` to `Any number of IntegrationTestScenarios` to clarify the statement.

Reply:
Done